db/schema.rb

```diff
@@ -49,11 +48,10 @@
     t.string   "nickname"
     t.boolean  "admin",                  default: false
     t.datetime "hidden_at"
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.